modules/search/instant-search/components/search-app.jsx

```diff
@@ -21,38 +21,53 @@ import {
 	getFilterQuery,
 	getSearchQuery,
 	getSortQuery,
+	getResultFormatQuery,
 	hasFilter,
 	restorePreviousHref,
 } from '../lib/query-string';
 import { removeChildren, hideElements, hideChildren, showChildren } from '../lib/dom';
 
 class SearchApp extends Component {
 	static defaultProps = {
-		resultFormat: 'minimal',
 		widgets: [],
 	};
 
 	constructor() {
 		super( ...arguments );
 		this.input = createRef();
-		this.state = { isLoading: false, response: {}, requestId: 0, showResults: false };
+		this.state = {
+			isLoading: false,
+			response: {},
+			requestId: 0,
+			showResults: false,
+		};
 		this.getResults = debounce( this.getResults, 200 );
 		this.prepareDomForMounting();
 	}
 
 	componentDidMount() {
-		this.getResults( getSearchQuery(), getFilterQuery(), this.props.initialSort, null );
+		this.getResults(
+			getSearchQuery(),
+			getFilterQuery(),
+			this.props.initialSort,
```

Comment:
It'd be nice if we could simplify the `determineDefaultSort` logic for `initialSort`. It's outside the scope of this PR, though.